py/server/deephaven/_wrapper.py

```diff
@@ -117,6 +119,39 @@ def _lookup_wrapped_class(j_obj: jpy.JType) -> Optional[type]:
     return None
 
 
+def javaify(obj: Union[JObjectWrapper, jpy.JType, Any]) -> Optional[jpy.JType]:
+    """
+    Returns an object that is safe to pass to Java. Callers should take care to ensure that this happens
+    in a liveness scope that reflects the lifetime of the reference to be passed to Java.
+    """
+    if obj is None:
+        return None
+    if isinstance(obj, JObjectWrapper):
+        return obj.j_object
+    if isinstance(obj, jpy.JType):
+        return obj
+    # We must return a java object, so wrap in a PyObjectLivenessNode so that the server's liveness tracking
+    # will correctly notify python that the object was released
+    return JPyObjectRefCountedNode(obj)
+
+
+def pythonify(j_obj: Union[jpy.JType, Any]) -> Optional[Union[JObjectWrapper, jpy.JType, Any]]:
```

Comment:
```suggestion
def pythonify(j_obj: Any) -> Optional[Union[JObjectWrapper, jpy.JType, Any]]:
```
redundant type hints?

Reply:
Ditto, typehints have no actual effect beyond "documentation and your IDE can look disapprovingly at you".